data/companies.json

```
@@ -544,6 +544,23 @@
         "youtube": null
       }
     },
+    {
+      "name": "Bytecare Technology",
+      "logoName": "bytecareTechnology.png",
+      "establishedAt": 1593998205,
+      "location": "Jawalakhel, Lalitpur",
+      "website": "https://www.bytecaretech.com/",
+      "email": "contact@bytecaretech.com",
+      "phone": "+977 9815303657,9849853141",
```

The format of the phone number is incorrect. If available, please also include a landline number.
```suggestion
      "phone": "+977-9815303657",
```